### PR TITLE
[submodule] Update submodule HEAD for sonic-mgmt-framework and sonic-telemetry

### PR DESCRIPTION
**Pull request by liushilongbuaa**
#### Why I did it
golang lib xmlquery v1.2.1 has critical security issue. MS Component Governance created an alert.
Update submodule HEAD to fix CG alert about CVE-2020-25614
#### How I did it
sonic-mgmt-framework
```
a72d9ee Fix CG alert CVE-2020-25614 about xmlquery v1.2.1 (#91)
```
sonic-telemetrey
```
727aefd Fix CG alert CVE-2020-25614 about xmlquery v1.2.1 (#107)
```
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

### CODE REVIEW

**Comment by lguohan** [comment]
@qiluo-msft , this is submodule update, can you check and approve?

File: .gitmodules [hunk 1]
```diff
@@ -80,7 +80,7 @@
 	url = https://github.com/sonic-net/sonic-mgmt-framework
 [submodule "src/sonic-telemetry"]
 	path = src/sonic-telemetry
-	url = https://github.com/sonic-net/sonic-telemetry
+	url = https://github.com/sonic-net/sonic-gnmi
```
Comment (yxieca) [inline .gitmodules:83]: This change doesn't look right.

**Review by yxieca — CHANGES_REQUESTED** [review]
Why are you linking telmetry to gnmi?

File: .gitmodules (hunk 1 above)
Comment (liushilongbuaa) [reply .gitmodules:83]: sonic-telemetry is archived. I pushed sonic-telemetry's code to sonic-gnmi's 202205 branch.

File: .gitmodules [hunk 2]
```diff
@@ -80,7 +80,7 @@
 	url = https://github.com/sonic-net/sonic-mgmt-framework
 [submodule "src/sonic-telemetry"]
```
Comment (qiluo-msft) [inline .gitmodules:81]: The submodule name is still sonic-telemetry. Is it easy to change this one in the same PR?

File: .gitmodules [hunk 3]
```diff
@@ -80,7 +80,7 @@
...
 	path = src/sonic-telemetry
```
Comment (qiluo-msft) [inline .gitmodules:82]: The path is still sonic-telemetry. Is it easy to change this one in the same PR?

Comment (liushilongbuaa) [reply .gitmodules:82]: If path changed, rules/telemetry.mk will not work.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**